### DISCUSSION TITLE
Can you explain screen layer where it is coming and how to create in my project using grapes?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4251] [Q&A]
Hai i have attached the image with green highlighted.. that screen layer i need to add in my custom block with screen id...
Can you explain how to create that one please?
![image (56)](https://user-images.githubusercontent.com/50009624/162886022-2e8fb253-9f03-4ade-9c4f-b1413b56d98e.jpg)

**Comment by artf** [comment]
There is no Screen component in the default demo.

![Schermata 2022-04-12 alle 16 08 13](https://user-images.githubusercontent.com/11614725/162981329-134c33ab-59c4-4460-b218-ef291bd48b91.png)

**Reply by senthil177cse** [reply]
oh ok... how to do import temeplate which is in default demo? and how to add component like header, section , footer like this?

**Reply by artf** [reply]
Here you can find the source of the demo project https://github.com/artf/grapesjs/blob/gh-pages/demo.html

**Reply by senthil177cse** [reply]
is the code is after generated HTML, CSS or while implement grapesjs itself?
